packages/inputs-minimal/src/radio/index.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import styles from './index.module.scss';
+
+const Radio = ({
+  onChange,
+  className,
+  disabled,
+  value,
+  label,
+  checked,
+  name,
+}) => {
+  const handleChange = (e) => {
+    onChange(value, e);
+  };
+
+  return (
+    <label className={clsx(styles.label, className, { [styles.disabled]: disabled })}>
+      <input
+        className={styles.input}
+        onChange={handleChange}
+        disabled={disabled}
+        type="radio"
+        checked={checked}
+        name={name}
+      />
+      <i className={styles.checkbox} />
+
+      {label && <div className={styles.labelText}>{label}</div>}
+    </label>
+  );
+};
+
+Radio.propTypes = {
+  value: PropTypes.string.isRequired,
+  onChange: PropTypes.func.isRequired,
+  name: PropTypes.string.isRequired,
```

Comment:
why is name required? you could use this control outside of an form - then there's no point in having a name